### PR TITLE
Profile menu: display signed-in user's information

### LINKED ISSUE #361: User cannot see who actually is logged in.
A user cannot see whether its him or another person who is logged in. This becomes problematic on shared devices.

The current website solves this issue by using [the topbar](https://developer.bcc.no/bcc-widgets/widgets/topbar.html).

### PR DESCRIPTION
**Pull request by sifferhans**
Closes #361 

CleanShot 2024-06-25 at 15 45 45@2x

### CODE REVIEW

File: components/ProfileMenu.vue [hunk 1]
```diff
@@ -61,15 +61,16 @@ const saveAndCloseContentLanguageDialog = () => {
 
 const { data: user } = useCurrentUser();
 </script>
+
 <template>
-  <div>
+  <div v-if="auth0.user.value">
```
Comment (andreasgangso) [inline components/ProfileMenu.vue:66]: maybe destructure auth0

**Review by andreasgangso — CHANGES_REQUESTED** [review]

**Comment by sifferhans** [comment]
Looks like this now, as Søren designed in #361 

https://github.com/bcc-code/bmm-web/assets/18753964/6a9aea2b-1ead-4626-94b4-03971dff4485

File: components/dropdown/DropdownMenuItem.vue [hunk 2]
```diff
@@ -18,29 +18,31 @@ const component = computed(() =>
...
-    </component>
+    <slot name="item" :active="active">
+      <component
+        :is="component"
+        :to="to"
+        :href="href"
+        :class="{
+          'bg-label-separator': active,
+        }"
+        class="type-subtitle-2 flex w-full items-center justify-between rounded-lg px-3 py-2 text-left"
```
Comment (kkuepper) [inline components/dropdown/DropdownMenuItem.vue:29]: can you add whitespace-normal. That prevents a blowout if you have many content languages

**Review by kkuepper — CHANGES_REQUESTED** [review]
prevent content languages from destroying the layout

**Review by kkuepper — CHANGES_REQUESTED** [review]
whitespace-normal didn't have the desired effect. Sorry.

File: components/dropdown/DropdownMenuItem.vue (hunk 2 above)
Comment (sifferhans) [reply components/dropdown/DropdownMenuItem.vue]: @kkuepper Does this look good? I added `truncate` to the `DropdownMenuItem` and added a max-width to the `DropdownMenu`.

CleanShot 2024-06-26 at 18 39 11@2x

Comment (kkuepper) [reply components/dropdown/DropdownMenuItem.vue]: That works

**Review by kkuepper — APPROVED** [review]

**Pull request merged**